presto-main/src/main/java/com/facebook/presto/Session.java

```diff
@@ -327,11 +327,9 @@ public Session beginTransactionId(TransactionId transactionId, TransactionManage
         for (Entry<String, SelectedRole> entry : identity.getRoles().entrySet()) {
             String catalogName = entry.getKey();
             SelectedRole role = entry.getValue();
-
```

Comment:
This commit looks does nothing . But let's keep consistent with PrestoSQL ;)

Reply:
What commit is that?

Reply:
Found it: `Store catalog selected roles in Identity`. Let me drop it.

Reply:
I see. Sorry for forgetting to use "Add Review" mode...